doc/source/serve/getting_started.md

```diff
@@ -65,339 +71,283 @@ PyTorch, and Tensorflow for more info and examples:
+In this section, we'll deploy the text translation model using Ray Serve, so
+it can be scaled up and queried over HTTP. We'll start by converting
+`Translator` into a Ray Serve deployment that runs locally on your computer.
 
-We start by opening a new Python file. First, we need to import `ray` and
-`ray serve`, to use features in Ray Serve such as `deployments`, which
-provide HTTP access to our model.
+First, we open a new Python file and import `ray` and `ray serve`:
 
-```{literalinclude} ../../../python/ray/serve/examples/doc/e2e_deployment.py
+```{literalinclude} ../serve/doc_code/getting_started/model_deployment.py
+:start-after: __import_start__
 :end-before: __import_end__
 :language: python
-:start-after: __import_start__
 ```
 
-After these imports, we can include our model code from above.
-We won't call our `summarize` function just yet though!
-We will soon add logic to handle HTTP requests, so the `summarize` function
-can operate on article text sent via HTTP request.
+After these imports, we can include our model code from above:
 
-```{literalinclude} ../../../python/ray/serve/examples/doc/e2e_deployment.py
-:end-before: __local_model_end__
+```{literalinclude} ../serve/doc_code/getting_started/model_deployment.py
+:start-after: __model_start__
+:end-before: __model_end__
 :language: python
-:start-after: __local_model_start__
 ```
 
-Ray Serve needs to run on top of a Ray cluster, so we connect to a local one.
-See {ref}`serve-deploy-tutorial` to learn more about starting a Ray Serve
-instance and deploying to a Ray cluster.
+The `Translator` class has two modifications:
+1. It has a decorator, `@serve.deployment`.
+2. It has a new method, `__call__`.
 
-```{literalinclude} ../../../python/ray/serve/examples/doc/e2e_deployment.py
-:end-before: __start_ray_cluster_end__
-:language: python
-:start-after: __start_ray_cluster_start__
-```
+The decorator converts `Translator` from a Python class into a Ray Serve
+`Deployment` object.
 
-The `address` parameter in `ray.init()` connects your Serve script to a
-running local Ray cluster. Later, we'll discuss how to start a local Ray
-cluster.
+Each deployment stores a single Python function or class that you write and uses
+it to serve requests. You can scale and configure each of your deployments independently using
+parameters in the `@serve.deployment` decorator.
 
-:::{note}
-`ray.init()` connects to or starts a single-node Ray cluster on your
-local machine,  which allows you to use all your CPU cores to serve
-requests in parallel. To start a multi-node cluster, see
-{ref}`serve-deploy-tutorial`.
-:::
+Deployments receive Starlette HTTP `request` objects [^f1]. If your deployment stores a Python function, the function is called on this `request` object. If you deployment stores a class, the class's `__call__` method is called on this `request` object. The return value is sent back in the HTTP response body.
```

Comment:
Can we put the explanation like **note** section?

like
<img width="800" alt="image" src="https://user-images.githubusercontent.com/6515354/181860794-3821275e-f682-4d5f-9be8-ff3427f6e778.png">


And then user can easily find  ```Next, we need to...```